packages/site/src/data/react/libraries.ts

```diff
@@ -123,13 +123,12 @@ export const libraries: Library<typeof libraryTags[number]>[] = [
 	{
 		name: "React Query",
 		author: "Tanner Linsley",
-		gitHubRepo: "tannerlinsley/react-query",
+		gitHubRepo: "TanStack/react-query",
```

Comment:
Little nitpick here: could you change this to `TanStack/query` instead 🙏 ?

Reply:
@jesus4497 I pushed the change. Can you help to check it again?